docs/SUMMARY.md

```diff
@@ -1,14 +1,17 @@
 # Summary
 
-* [Introduction](README.md)
-* [Getting started](getting_started/README.md)
-    * [An HTTP Server](getting_started/http_server.md)
-    * [A Command Line Application](getting_started/cli.md)
-* [Using the compiler](using_the_compiler/README.md)
-* [The shards command](the_shards_command/README.md)
-* [About this guide](overview/README.md)
-* [Syntax and semantics](syntax_and_semantics/README.md)
+* [Home](README.md)
+    * [Getting started](getting_started/README.md)
+        * [An HTTP Server](getting_started/http_server.md)
+        * [A Command Line Application](getting_started/cli.md)
+    * [Using the compiler](using_the_compiler/README.md)
+    * [The shards command](the_shards_command/README.md)
+    * [Coding style](conventions/coding_style.md)
+    * [Platform Support](platform_support.md)
+* [Specification](syntax_and_semantics/README.md)
```

Comment:
```suggestion
* Specification
    * [Syntax and semantics](syntax_and_semantics/README.md)
```

Reply:
Now *this* landing page also qualifies for "no useful content" :D

#480 doesn't really change that, either.

Reply:
If this PR lands first, then definitely delete that page in #480